code/RedirectedURLCsvBulkLoader.php

```diff
@@ -18,7 +16,7 @@ class RedirectedURLCsvBulkLoader extends CsvBulkLoader
      */
     protected function processRecord($record, $columnMap, &$results, $preview = false) {
         // make $exception_thrown false initially
-        $this->exception_thrown = false;
+        $exception_thrown = false;
```

Comment:
can we use our general coding style, please? ie: `$exceptionThrown`

Reply:
Done